src/components/common/PeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -211,6 +273,18 @@ export default class PeopleAndRoles extends Mixins(CommonMixin) {
       'This undo will restore the original Completing Party.'
   }
 
+  private get orgPersonLabel (): string {
+    return this.getResource.changeData?.orgPersonInfo.orgPersonLabel
+  }
+
+  private get orgPersonSubtitle (): string {
+    return this.getResource.changeData?.orgPersonInfo.subtitle
+  }
+
+  private get orgPersonHelp (): HelpSectionIF {
```

Comment:
I've been changing my gets to protected since the template uses them. For these, it wouldn't matter if they were public (which is the default if not specified) since code external to this class could safely call them. Just food for thought.

Reply:
(In VS Code, private variable/getters/methods show as "unused" when they are only used in the template.)

Reply:
I haven't noticed this in Webstorm but it is probably due to some configuration i have?

Reply:
Works either way atm (looks like TS doesn't enforce visibility rules with the template). Resolving.